app/views/games/index.html.erb

```diff
@@ -3,6 +3,6 @@
 <%= link_to "New Game", new_game_path %>
 
 <% @games.each do |game| %>
-  <p><%= link_to game.name, game_path(game) %></p>
+  <p><%= link_to game.name, add_player(game) %></p>
```

Comment:
Instead of a link, you could use a form tag with a submit button, that submits data to the `update` action.